newproject01/newproject01/q3.c

```diff
@@ -0,0 +1,63 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define stringlen 256
+
+int isJumble(char*);
+
+main()
+{
+	char string[stringlen], longstring[stringlen], end[] = "end", End[] = "End";
+	int long_s_size = 0, templength;
+
+	printf("write a sentence(series of strings)end with ''End'' please: ");
+	if (scanf("%s", string) != 1)
+		return 1;
+	long_s_size = strlen(string);
+
+	while (strcmp(string, End) != 0)
+	{
+		if (isJumble(string))
+		{
+			if (strlen(string) > long_s_size)
+			{
+				strcpy(longstring, string);
```

Comment:
change longstring to snake_case. also, use better name, for example `longest_string`.